tests/virt_autotest/install_package.pm

```diff
@@ -76,7 +76,17 @@ sub install_package {
     #install qa_lib_virtauto
     if (check_var('ARCH', 's390x')) {
         lpar_cmd("zypper --non-interactive --gpg-auto-import-keys ref");
-        lpar_cmd("zypper --non-interactive in qa_lib_virtauto");
+        my $pkg_lib_data = "qa_lib_virtauto-data";
+        my $cmd          = "rpm -qa | grep $pkg_lib_data";
```

Comment:
This should be faster as it won't iterate over all installed RPMs:
```suggestion
        my $cmd          = "rpm -q $pkg_lib_data";
```
My assumption here is that `$pkg_lib_data` has the complete RPM name.

Reply:
Got it, had changed code as your mentioned. Thank you so much for your great help.